helm-chart/csi-driver/values.yaml

```diff
@@ -18,7 +18,11 @@ namespace: kube-system
 # "linode-block-storage" or left as an empty string
 defaultStorageClass: linode-block-storage-retain
 
-# Images - Default
+# set these value to a comma seperated string if you'd like to add tags to the created volumes
+# volumeTags:
```

Comment:
nit: might be a nicer interface for a helm chart to do a list and `{{ join "," .Values.volumeTags }}` ?

```
volumeTags:
 - tag1
 - tag2
```

Reply:
That would be nice..

Reply:
done - updated